patterns/template-single-photo.php

```diff
@@ -0,0 +1,90 @@
+			<!-- wp:post-title {"level":1,"fontSize":"x-large"} /-->
+			<!-- wp:group {"style":{"spacing":{"blockGap":"var:preset|spacing|50"}},"layout":{"type":"flex","orientation":"vertical"}} -->
+			<div class="wp-block-group">
+			<!-- wp:group {"style":{"spacing":{"blockGap":"var:preset|spacing|10"}},"layout":{"type":"constrained"}} -->
+			<div class="wp-block-group">
+				<!-- wp:paragraph {"fontSize":"x-small"} -->
+				<p class="has-x-small-font-size">Published on</p>
+				<!-- /wp:paragraph -->
+				<!-- wp:post-date {"style":{"elements":{"link":{"color":{"text":"var:preset|color|contrast"}}}},"textColor":"contrast","fontSize":"x-small"} /-->
+			</div>
+			<!-- /wp:group -->
+			<!-- wp:group {"style":{"spacing":{"blockGap":"var:preset|spacing|10"}},"layout":{"type":"constrained"}} -->
+			<div class="wp-block-group">
+				<!-- wp:paragraph {"fontSize":"x-small"} -->
+				<p class="has-x-small-font-size">Posted by</p>
+				<!-- /wp:paragraph -->
+				<!-- wp:post-author-name {"isLink":true,"fontSize":"x-small"} /-->
+			</div>
+			<!-- /wp:group -->
+			</div>
+			<!-- /wp:group -->
+			<!-- wp:group {"style":{"spacing":{"blockGap":"var:preset|spacing|50"}},"layout":{"type":"flex","orientation":"vertical"}} -->
+			<div class="wp-block-group">
+				<!-- wp:group {"style":{"spacing":{"blockGap":"var:preset|spacing|20"}},"layout":{"type":"constrained"}} -->
+				<div class="wp-block-group">
+					<!-- wp:paragraph {"fontSize":"x-small"} -->
+					<p class="has-x-small-font-size">Categories:</p>
+					<!-- /wp:paragraph -->
+					<!-- wp:post-terms {"term":"category","style":{"typography":{"fontStyle":"normal","fontWeight":"400"}}} /-->
+				</div>
+				<!-- /wp:group -->
+				<!-- wp:group {"style":{"spacing":{"blockGap":"var:preset|spacing|20"}},"layout":{"type":"constrained"}} -->
+				<div class="wp-block-group">
+					<!-- wp:paragraph {"fontSize":"x-small"} -->
+					<p class="has-x-small-font-size">Tagged:</p>
+					<!-- /wp:paragraph -->
+					<!-- wp:post-terms {"term":"post_tag","style":{"typography":{"fontStyle":"normal","fontWeight":"400"}}} /-->
+				</div>
+				<!-- /wp:group -->
+			</div>
+			<!-- /wp:group -->
+		</div>
+		<!-- /wp:group -->
+
+		<!-- wp:group {"align":"full","style":{"spacing":{"padding":{"right":"var:preset|spacing|50","left":"var:preset|spacing|50"},"margin":{"top":"var:preset|spacing|50","bottom":"0"}}},"layout":{"type":"default"}} -->
+		<div class="wp-block-group alignfull" style="margin-top:var(--wp--preset--spacing--50);margin-bottom:0;padding-right:var(--wp--preset--spacing--50);padding-left:var(--wp--preset--spacing--50)">
+			<!-- wp:group {"tagName":"nav","align":"full","style":{"spacing":{"padding":{"top":"var:preset|spacing|40","bottom":"var:preset|spacing|40"},"margin":{"top":"0","bottom":"0"}}},"layout":{"type":"flex","flexWrap":"nowrap","justifyContent":"space-between"}} -->
```

Comment:
I tried setting the small font size directly on the group, but it did not work, the post navigation link did not inherit the size from the group. So the font size is added once for each block.